src/data/dateGenerator.js

```diff
@@ -0,0 +1,25 @@
+export function createCategoryElement(url, element) {
+  return {url, element};
+}
+
+export function makeStatus(hasStatus, statuses) {
+  return {hasStatus, statuses};
+}
+
+export function makeOption(actualValue, showedValue) {
+  return {actualValue, showedValue}
+}
+
+export function generateRandomString(coupons, stringLength) {
+  const randomStrings = [];
+  for (let i = 0; i < coupons; i++) {
+    const randomString = Math.random()
+      .toString(36)
+      .substring(2, 2 + stringLength);
+    if (randomStrings.includes(randomString)) {
+      continue;
+    }
+    randomStrings.push(randomString);
+  }
+  return randomStrings;
+}
```

Comment:
```suggestion
export const generateRandomStringArray = (arrayLength, stringLength) => {
  const randomStrings = new Set();
    
  while (randomStrings.size !== arrayLength) {
    const randomString = Math.random()
      .toString(36)
      .substring(2, 2 + stringLength);
    randomStrings.add(randomString);
  }
    
  return [ ...randomStrings ];
}
```
오우 절대 그럴일은 없겠지만 만약 생성된 스트링이 이미 배열에 존재한다면 안넣고 넘어가겠네요.
Set을 사용하면 이미 존재하는지 확인하는 로직을 작성하지 않아도 될 것 같습니다. Set은 유니크한 요소만 가지고 있으니까 만약 Set의 size가 변하지 않았다면 방금 추가한 요소는 이미 중복으로 갖고 있었다는 얘기가 되겠죠.

또한 화살표 함수로 바꾸는 것도 좋을 것 같네요. 사실 별 차이는 없지만 화살표 함수가 일반 함수보다 더 가볍습니다. this 바인딩도 없고 constructor도 없거든요.

함수 이름과 매개변수 이름도 좀 더 명확하게 바꾸는게 좋을 것 같아요. 쿠폰과 상관없이 사용될 수 있는 함수니까요

Reply:
화살표 함수 성애자로써 참 뿌듯하네요.

Reply:
오... 이런 피드백 너무 좋아요😁

Reply:
const 로 바꾸면 실행 값을 바로 `generateRandomStringArray `에 할당하나보넹? 코틀린에도 같은 기능이 있는디